extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -741,6 +762,34 @@ export class AmpA4A extends AMP.BaseElement {
   }
 
   /**
+   * Refreshes ad slot by fetching new creative and rendering it.
+   */
+  refresh() {
+    this.isRefreshing_ = true;
+    this.tearDownSlot();
```

Comment:
Option 2 is what we initially planned on, so that's what I'm going to implement.